content/authoring/tutorials/create-first-kata.mdx

```diff
@@ -97,110 +97,127 @@ This is where you specify how difficult you think your kata is. This value is us
-
-Note that similar principles apply to interpreted languages (e.g. JavaScript, Python, Ruby): the initial solution should not contain syntax errors and/or (top-level) reference errors which may prevent the solver from getting started with the task immediately.
+The initial solution is the code users are provided with when they start training on your kata. How you set up your initial solution code will depend heavily on the discipline you selected. Bug-fixing and refactoring disciplines require the inclusion of a partially or fully working solution respectively within this block. For fundamentals and algorithm disciplines you will likely only include skeleton code, such as an empty function, class or other code that serves as a starting point for a user's solution. In some cases you may just want to include a few helpful comments to help get the user started, but no actual code itself.
+
+Special care must be taken when providing code for the initial solution. You may have encountered a situation yourself where the provided initial solution was incorrect (e.g. code that wouldn't compile, function signatures that differed from the tests or description, etc.). Hunting down the required information in the description or sample tests is an exercise in frustration and, more importantly, a waste of time that should have been spent actually *solving* the kata.
+
+The _only time_ non-compiling code is acceptable is when it is an _intrinsic part if the challenge_, for example when the kata focuses explicitly on debugging or involves solving language-specific compile-time problems. In any other case, **it is considered a kata issue of the initial solution does not compile**. To avoid this, you must obviously supply some kind of placeholder code or "dummy" implementation which, depending on the chosen language, typically takes the form of a no-op or return statement. Note that this placeholder code _only needs to satisfy the compiler_, but may (and likely will) lead to run-time errors if not replaced by the user.
+
+The same rule applies to interpreted languages (e.g. JavaScript, Python, Ruby): the initial solution must not contain syntax errors and/or (top-level) reference errors that prevent the user from immediately implementing their solution.
+
+Examples:
+```CSharp
+// C#
+public class Kata
+{
+    public static string RepeatString(string s, int n)
+    {
+        // your code goes here
+        return "";
+    }
+}
+```
+```Python
+# Python
+def repeat_string(s, n):
+    # your code goes here
+    pass
+```
```

Comment:
Example code added for illustration purposes, to show what a "correctly compiling initial solution" looks like.